### PR TITLE
Webapp folder structure proposal

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

**Comment by minotogna** [comment]
@sorja components name upper case yes, folder name it's something I would't like to see :p

**Comment by SteRiccio** [comment]
> @sorja components name upper case yes, folder name it's something I would't like to see :p

I agree with Mino about the folder names, even in the documentation of React the folder names start with a lowercase letter, but it depends... if the folder is to group the files of a component and there is an index.js file inside of it, then it makes sense to have it starting with an uppercase letter to harmonize the imports.

File: doc/webapp.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                    - surveyForm
+                    - taxonomies
+                    - taxonomy
+    - store
+        - app
+            - dialogConfirm
+            - errors
+            - job
+            - notification
+            - i18n
```
Comment (ramirobg94) [inline doc/webapp.md:58]: I am not sure it is the moment and the right place,

 I saw in the repo that all of the translations are into the same file, how about split it based on the view or context?

- i18n
- -  en
- - - Survey
- - - Designer
- - it
- - - ...

**Comment by ramirobg94** [comment]
I think that @SteRiccio comment is a good point, I am okay in both cases but this approach works for me. Maybe this brings up what should be or not into each component folder:

components/views (as we talked this morning)
ComponentName.js
style.scss
index.js

or

components
ComponentName.js
style.scss

File: doc/webapp.md (hunk 1 above)
Comment (minotogna) [reply doc/webapp.md:58]: yes I know, it's already in the refactor work plan. here only webapp code refactor

**Pull request merged**